helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -113,6 +114,7 @@
   private final boolean _usesExternalZkClient;
 
   private static Logger logger = LoggerFactory.getLogger(ZKHelixAdmin.class);
+  private static final long VALUE_NOT_SET = -1;
```

Comment:
sorry if i didnt make myself more clear. when i said `VALUE_NOT_SET`, we usually name these constants in a more descriptive manner - e.g.) `OFFLINE_DURATION_VALUE_NOT_SET` for example. this is ZKHelixAdmin, so it contains a lot of different operations, so it's better to make this crystal clear.

also nit: usually `long`s have an `L` appended to it. -> `-1L;`

Reply:
Interesting, in the other PR, there's a comment asking for generalize the NOT SET value, and that's why I modified it. I separated them into different definitions again. Let me know whether this is good.

Reply:
@zhangmeng916 I actually agree with that suggestion that we should combine all NOT SET values into one. Sorry I wasn't aware that there were many other NOT SET bits floating around.

It's either or - you could combine all of them into one (just make sure they're all using -1) or leave them all separate and create a separate issue/PR for it.

Reply:
I will create a new PR for standardize the NOT SET value. Seems there are a lot, and I'll modify together.